File: flutter_demo/lib/demo/component_creation.dart
import 'package:flutter/cupertino.dart';

class ComponentCreation extends StatelessWidget {
  const ComponentCreation({super.key});

  @override
  Widget build(BuildContext context) {
    return const Padding(
      padding: EdgeInsets.all(8.0),
      child: Text('Please select a type from the navigation bar to see effects.'),
    );
  }
}

File: ComposeDemo/app/src/main/java/com/example/composedemo/demo/ComponentCreation.kt
package com.example.composedemo.demo

import androidx.compose.foundation.layout.padding
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.unit.dp

@Composable
fun ComponentCreation() {
    Text(
        text = "Please select a type from the navigation bar to see effects.",
        modifier = Modifier.padding(all = 16.dp)
    )
}